src/index.js

```diff
@@ -895,10 +909,17 @@ BigQuery.prototype.createJob = function(options, callback) {
         return;
       }
 
+      if (resp.status.errors) {
+        err = new common.util.ApiError({
+          errors: resp.status.errors,
+          response: resp,
+        });
+      }
+
       var job = self.job(jobId);
       job.metadata = resp;
 
-      callback(null, job, resp);
+      callback(err, job, resp);
```

Comment:
So we should still return a job even though we returned an error?

Reply:
I think I want some one on the BQ team to weigh in here (@tswast @jjolma). I'm not sure if an error is present in the job status that it means the job should be discarded.

Reply:
Yes, the job should be available even on errors. There are sometimes some useful info for debugging in the `statistics` property of the job.